packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts

```diff
@@ -34,35 +34,57 @@ import {
   COMPOSE_EXTENSIONS_TPL_LOCAL_DEBUG,
   COMPOSE_EXTENSIONS_TPL_EXISTING_APP,
   TEAMS_APP_SHORT_NAME_MAX_LENGTH,
+  MANIFEST_TEMPLATE_CONSOLIDATE,
 } from "./constants";
 import { replaceConfigValue } from "./utils/utils";
 
 export async function getManifestTemplatePath(
   projectRoot: string,
-  isLocalDebug: boolean
+  isLocalDebug = false
 ): Promise<string> {
   const appDir = await getAppDirectory(projectRoot);
-  return isLocalDebug ? `${appDir}/${MANIFEST_LOCAL}` : `${appDir}/${MANIFEST_TEMPLATE}`;
+  if (isConfigUnifyEnabled()) {
+    return `${appDir}/${MANIFEST_TEMPLATE_CONSOLIDATE}`;
+  } else {
+    return isLocalDebug ? `${appDir}/${MANIFEST_LOCAL}` : `${appDir}/${MANIFEST_TEMPLATE}`;
+  }
 }
 
-export async function init(projectRoot: string, appName: string): Promise<Result<any, FxError>> {
+export async function init(
+  projectRoot: string,
+  appName: string,
+  existingApp: boolean
+): Promise<Result<any, FxError>> {
   const newAppPackageFolder = `${projectRoot}/templates/${AppPackageFolderName}`;
   await fs.ensureDir(newAppPackageFolder);
 
-  let localManifestString = TEAMS_APP_MANIFEST_TEMPLATE_LOCAL_DEBUG_V3;
-  const suffix = "-local-debug";
-  let localAppName = appName;
-  if (suffix.length + appName.length <= TEAMS_APP_SHORT_NAME_MAX_LENGTH) {
-    localAppName = localAppName + suffix;
-  }
-  localManifestString = replaceConfigValue(localManifestString, "appName", localAppName);
-  const localManifest = JSON.parse(localManifestString);
-  await saveManifest(projectRoot, localManifest, true);
+  if (isConfigUnifyEnabled()) {
+    const manifestString = TEAMS_APP_MANIFEST_TEMPLATE_V3;
+    const manifest = JSON.parse(manifestString);
+    if (existingApp) {
+      manifest.developer = {
```

Comment:
Is this all needed for existing app?

Reply:
For existing app, it has different developer config:
![image](https://user-images.githubusercontent.com/71362691/156113979-c55a0e39-af6e-4e5f-950a-a916b7cc2097.png)

Reply:
When will tab content updated to the template?